libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -230,12 +230,25 @@ func (m *Manager) GetPids() ([]int, error) {
 	return cgroups.GetPids(dir)
 }
 
+// absClean cleans a path such that it can be safely appended to any path and
+// the resulting path will not escape the prefixed path. This is done entirely
+// lexically, with no filesystem calls. Thus, it does not account for symlinks
+// (which can cause security problems).
+func absClean(path string) string {
+	path = filepath.Clean(string(os.PathSeparator) + path)
+	path, _ = filepath.Rel(string(os.PathSeparator), path)
+	return path
+}
```

Comment:
If `path = /a/b/c`, `absClean(path)` will end up with `a/b/c`, seems not correct.